### DISCUSSION TITLE
How can I get training information live while training the model ?

### DISCUSSION THREAD

**Discussion by SJavad** [ultralytics/yolov5#11437] [Q&A]
I designed a UI application for training the yolov5 model and I wanna get parameters (such as mAP, loss, epoch, Precession, Recall, and...) while training.
if fact I want to show these parameters live on the application.
Do you know how I can do that? 
Would anyone be able to give me an idea?

**Comment by AyushExel** [comment]
Could you achieve that by using experiment tracking tools? We currently support various tracking tools that log experiment metrics in real time.

**Reply by SJavad** [reply]
thank's for your response.
I do not have a specific tracking tool in mind and I am not familiar with these types of tools.
I would be happy if you could introduce me to some of these tools and how to work with them in this particular context.
@AyushExel

**Reply by SJavad** [reply]
> Could you achieve that by using experiment tracking tools? We currently support various tracking tools that log experiment metrics in real time.

I found some tracking tools in yolov5 repository, but these tools work online and need internet connection or save experiment data after training process
I want to save experiment data on every epoch and get these training metrics and parameters live
how can I do that ?

**Comment by pinnintipraneethkumar** [comment]
HI @SJavad did you achieve this ? if so can you please guide me how did you do that.

Thank you
